pkg/tracker/timeOut.go

```diff
@@ -46,13 +48,16 @@ func (b *TimeOutTracker) Exec(ctx context.Context) error {
 
 	instance, err := getter.NewTellorGetters(contractAddress, client)
 	if err != nil {
-		fmt.Println("instance Error, disputeStatus")
+		level.Error(logger).Log("msg", "instance Error, disputeStatus", "err", err)
 		return err
 	}
 	address := "000000000000000000000000" + _fromAddress[2:]
 	decoded, err := hex.DecodeString(address)
 	if err != nil {
-		log.Fatal(err)
+		level.Error(logger).Log("msg", "error decoding address", "err", err)
+		os.Exit(1)
+		//Does log.Fatal terminates the execution automatically?
```

Comment:
yes that is why in practise here we shouldn't exit but just pass propagate the error to the caller.

Reply:
this still stands, 
would you mind letting me click the resolve button? 
This way it is easier for me to not miss any of the suggestions and changes. 
If you think that some suggestion is incorrect please comment back and will discuss.

Reply:
this one I don't understand what I'm supposed to change? Should I add  `os.Exit(1)` on all thE errors on this file or just return `errors.Wrapf(...)`?

Reply:
yes return `errors.Wrap`

Exit should happen only in the main package. All other errors should be either logged(when not that important) or returned to the caller